styles/styles.css

```diff
@@ -38,8 +39,8 @@
   /* heading sizes */
   --heading-font-size-xxl: 24px;
   --heading-font-size-xl: 21px;
-  --heading-font-size-l: 14px;
-  --heading-font-size-m: 12px;
+  --heading-font-size-l: 21px;
+  --heading-font-size-m: 14px;
```

Comment:
those values were matched with the current website online, how come these changed?

Reply:
@RemoLiechti - I think it is possible that some of these values are not consistent across all pages. Could you have a look and find the place you got the original values from?

Reply:
See my message about this on Friday :) it is not consistent, however we didn't use size-m yet, which I think is a way out